### DISCUSSION TITLE
irregular flow

### DISCUSSION THREAD

**Discussion by CptBarbo** [Jyers/Marlin#734] [Q&A] [answered]
![183428141_10160371657543532_7246000004943414026_n](https://user-images.githubusercontent.com/83835052/117492159-e272a580-af79-11eb-8c8a-91a969aa78c2.jpg)
I have a problem after upgrading(Jyers 1.3.2) Ender3 V2 4.2.7 no bltouch
My e step setting is constantly getting distorted and i have irregular flow. 
What can i do ?

**Comment by marcomania75** [comment]
![IMG_0697](https://user-images.githubusercontent.com/6855604/117534078-2e603180-aff0-11eb-9c92-3659724c9747.jpeg)
hmm, mine started underextruding after installing the new firmware so I changed/calibrated the e-steps for the extruder again. 

And it does it again now.

Maybe a similar reason?

**Reply by CptBarbo** [reply]
probably.  Because I am doing step calibration again, it is broken.  I am using 413mm in my original software but I had to use 430-450 in 1.3.2.

**Comment by CptBarbo** [comment, marked as answer]
In addition, the extruder moves very slowly @Jyers

**Reply by Jyers** [reply]
Could be a base marlin issue, I haven't modified anything extruder related recently

**Reply by marcomania75** [reply]
in my case i have the solution, but it will not be the solution for CptBarbo. After endless calibrations I measured the filaments diameter. It is too low, around 1.66-1.71mm. After setting the feedrate at 110-115% I have a nice Layer again...

**Comment by smooreace** [comment]
So what brand should the entire world be avoiding there?

**Reply by marcomania75** [reply]
i am trying to contact the (european) manufacturer and want to wait for their response before i tell.

**Reply by marcomania75** [reply]
extrudr will sent me a new spool,...
